test/composeStores.spec.js

```diff
@@ -27,5 +27,29 @@ describe('Utils', () => {
 
       expect(Object.keys(store({}, {type: 'push'}))).toEqual(['stack']);
     });
+    it('should throw an error if undefined return from store', () => {
+      const store = composeStores({
+        stack: (state = []) => state,
+        bad: (state = [], action) => {
+          if (action.type === 'something') {
+            return state;
+          }
+        }
+      });
+      expect(() => store({}, {type: '@@testType'})).toThrow();
+    });
+    it('should throw an error if undefined return not by default', () => {
+      const store = composeStores({
+        stack: (state = []) => state,
+        bad: (state = 1, action) => {
+          if (action.type === 'something') {
+            return undefined;
```

Comment:
I thought about doing that, but I wasn't sure the stance of this repo. I would also need to change it in composeStores.js also. What are people's thoughts?